user_guide_src/source/incoming/routing/049.php

```diff
@@ -1,3 +1,7 @@
 <?php
 
+// In app/Config/Routing.php
+public boolean $translateURIDashes = true;
```

Comment:
This is a syntax error. If you use `*.php` file, write correct PHP code.
The same applies to the following php files.

Reply:
Why do we enforce this linting in the user guide folder? Seems a bit pedantic....

```suggestion
$translateURIDashes = true;
```

Reply:
To check and fix sample code style, we are using `*.php` files and check them.